app/controllers/admin/positions_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Admin::PositionsController < ApplicationController
+  before_action -> { user_has_position(Position.first) }
+
+  def update
+    position = Position.find(params[:id])
+    if position.update(position_params)
+      flash[:notice] = "Event Type Admins updated successfully"
+    else
+      flash[:alert] = "Event Type Admins updated successfully"
+    end
+    redirect_to update_users_path
+  end
+
+  def create
+    position = Position.new(position_params)
+    if position.save
+      flash[:notice] = "\"#{position.name}\" created successfully"
+    else
+      flash[:alert] = "\"#{position.name}\" was not created successfully"
+    end
+    redirect_to update_users_path
+  end
+
+  def destroy
+    position = Position.find(params[:id])
+    default_positions = Position.where('id < 5')
```

Comment:
This hardcoded id is prob a bad idea (cause IDs can easily have the potential to be different). Maybe you should create a method in the position model that finds them by name and returns an array and then you can verify that way.